google/cloud/storage/blob.py

```diff
@@ -1750,7 +1749,7 @@ def _do_multipart_upload(
 
             This private method does not accept ConditionalRetryPolicy values
             because the information necessary to evaluate the policy is instead
-            evaluated in client.download_blob_to_file().
+            evaluated in blob._do_upload().
```

Comment:
Oh, it looks like we're evaluating it BOTH in blob._do_upload() and in client.download_blob_to_file(). This is harmless, which is why the tests didn't catch it, but we only need one of those.

I don't have a strong opinion on whether we do this in _do_upload or download_blob_to_file. Let's keep this docstring change and additionally we can remove the conditional retry policy code in client.download_blob_to_file, either in another PR, or in this one if you prefer.

Reply:
IIUC, I believe we'll need to remain both ConditionalRetryPolicy evaluations as one is for **upload** operations and the other is for **downloads**. I revised the docstrings here to match particularly where we're doing the evaluations for upload operations, `blob._do_upload()` .